shared/test/test_net_sim_api.rb

```diff
@@ -0,0 +1,55 @@
+require 'minitest/autorun'
+require 'rack/test'
+require File.expand_path '../../../deployment', __FILE__
+require File.expand_path '../../middleware/net_sim_api', __FILE__
+
+ENV['RACK_ENV'] = 'test'
+
+class NetSimApiTest < Minitest::Unit::TestCase
+
+  def create_record(record)
```

Comment:
Made test helpers private in [this commit](https://github.com/code-dot-org/code-dot-org/commit/80d5715d19ef3e07fd0d5937ba23c0812dd472d4).  Went ahead and applied this change to all of the test files in /shared.